README.md

```diff
@@ -7,38 +7,103 @@ sometimes referred to as a "meta operator" or an "operator for operators".
 Most importantly, this operator doesn't replace or interfere with OLM.
 It only creates operator CRs, which is the user's prerogative.
 
-## Using the HCO
 TODO:
-  - Golang code to generate deployment manifests
-  - Manifest that launches HCO, kubevirt, CDI, network, and UI operators for
-    initial non OLM deployments
-  - Unifed CSV file that lauches all operators through OLM
+- [x] Golang code to generate deployment manifests
+- [x] Unifed CSV file that lauches all operators through OLM
+- [ ] Use operator-courier to publish HCO CSV
+- [ ] Manifest that launches HCO, kubevirt, CDI, network, and UI operators for
+  initial non OLM deployments
+
+## Using the HCO
+
+Create the namespace for the HCO.
+```bash
+oc create ns kubevirt-hyperconverged
+```
+
+Switch to the HCO namespace.
+```bash
+oc project kubevirt-hyperconverged
+```
+
+Launch all of the CRDs.
+```bash
+oc create -f deploy/converged/crds/hco.crd.yaml
+oc create -f deploy/converged/crds/cdi.crd.yaml
+oc create -f deploy/converged/crds/kubevirt.crd.yaml
+```
+
+Launch all of the Service Accounts, Cluster Role(Binding)s, and Operators.
+```bash
+oc create -f deploy/converged
+```
+
+Create an HCO CustomResource, which creates the KubeVirt CR, launching KubeVirt.
+```bash
+oc create -f deploy/converged/crds/hco.cr.yaml
+```
+
+## Launching the HCO through OLM
+
+**NOTE**
+Until we determine how to publish (and consume) the HCO through
+operator-courier|Marketplace|operatorhub.io, this is a means to demonstrate the
+HCO workflow through OLM. Replace `<docker_org>` with your Docker organization
+as official operator-registry images for HCO will not be provided.
+
+Build and push the converged HCO operator-registry image.
 
-Create component operator namespaces.
 ```bash
-oc create ns kubevirt
-oc create ns cdi
+cd deploy/converged
```

Comment:
Let's use a variable for <docker_org>.  It will make consuming these commands easier.
```
export HCO_DOCKER_ORG=djzager
```